src/common/helpers.rs

```diff
@@ -30,6 +30,24 @@ pub fn create_search_runtime(max_search_threads: usize) -> std::io::Result<Runti
         .build()
 }
 
+pub fn create_optimizer_runtime(max_optimization_threads: usize) -> std::io::Result<Runtime> {
+    let mut optimize_runtime_builder = runtime::Builder::new_multi_thread();
+
+    optimize_runtime_builder
+        .enable_time()
```

Comment:
I have removed the previous constraint `.worker_threads(3)` due to the global nature of this runtime.